src/v/redpanda/admin/api-doc/debug_bundle.json

```diff
@@ -0,0 +1,49 @@
+"/v1/debug/bundle": {
+  "post": {
+    "description": "Posting to this endpoint will trigger the start of an rpk debug bundle process.  If a process is already executing, this endpoint will return a 409 error.  If there is an issue with starting the process, this endpoint will return a 500 error.\n\nNote if job_id is not provided in BundleStartConfig, one will be automatically generated and returned in BundleStartResponse.",
+    "summary": "Start an rpk debug bundle",
+    "operationId": "post_debug_bundle",
+    "consumes": [
+        "application/json"
+    ],
+    "parameters": [
+      {
+        "name": "config",
+        "in": "body",
+        "required": true,
+        "schema": {
+          "$ref": "#/definitions/bundle_start_config"
+        }
+      }
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "responses": {
+      "200": {
+        "description": "OK",
+        "schema": {
+          "ref": "#/definitions/bundle_start_response"
+        }
+      },
+      "401": {
+        "description": "Unauthorized",
+        "schema": {
+          "$ref": "#/definitions/error_body"
+        }
+      },
+      "403": {
+        "description": "Forbidden",
+        "schema": {
+          "$ref": "#/definitions/error_body"
+        }
+      },
+      "409": {
+        "description": "Conflict",
+        "schema": {
+          "$ref": "#/definitions/error_body"
+        }
```

Comment:
Are we missing a 422 for invalid configuration?

Reply:
> Are we missing a 422 for invalid configuration?
It's not in the API spec for GET, there is no supplied configuration to be invalid, and I can't see anywhere where that error could be produced.

Reply:
Oh weird ok that must have been it, yeah I thought i put the comment on the POST and not the GET.  Computers man

Reply:
> Oh weird ok that must have been it, yeah I thought i put the comment on the POST and not the GET. Computers man

You did. But the diff is greedy, so it "moved".